node/core/dispute-coordinator/src/scraping/mod.rs

```diff
@@ -177,6 +177,17 @@ impl ChainScraper {
 	/// As long as we have `MAX_FINALITY_LAG` this makes sense as a value.
 	pub(crate) const ANCESTRY_SIZE_LIMIT: u32 = MAX_FINALITY_LAG;
 
+	/// How many finalized block the scraper should process.
+	///
+	/// Initialize this value with `DISPUTE_CANDIDATE_LIFETIME_AFTER_FINALIZATION`
+	/// If  `SCRAPED_FINALIZED_BLOCKS_COUNT` > `DISPUTE_CANDIDATE_LIFETIME_AFTER_FINALIZATION`
+	/// scraper's job will be undone on pruning.
+	pub(crate) const SCRAPED_FINALIZED_BLOCKS_COUNT: NonZeroU32 =
```

Comment:
I don't see why these two values would ever need to differ. I would have just used `DISPUTE_CANDIDATE_LIFETIME_AFTER_FINALIZATION` directly, just changing docs. Name is actually still fine, I think.